v2/mongodb-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/mongodb/options/MongoDbToBigQueryOptions.java

```diff
@@ -67,6 +67,19 @@ public interface MongoDbOptions extends PipelineOptions, DataflowPipelineOptions
     String getUserOption();
 
     void setUserOption(String userOption);
+
+     @TemplateParameter.KmsEncryptionKey(
+      order = 5,
+      optional = true,
+      groupName = "Source Parameters",
+      description = "Google Cloud KMS key",
+      helpText =
+          "Cloud KMS Encryption Key to decrypt the username, password, and connection string. If Cloud KMS key is "
+              + "passed in, the username, password, and connection string must all be passed in encrypted.",
```

Comment:
I think here we mean the `connectionUri` only? Since I think everything is sent in that single param

Reply:
That's right. I've copied the parameter from the Jdbc and forgot to change the description.